hack/update-generated-docs.sh

```diff
@@ -25,6 +25,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+# This only runs on master
+[[ $(git rev-parse --abbrev-ref HEAD) == "master" ]] || exit
```

Comment:
I think a better fix is to run versionize docs after this script. I don't think disabling this is a good solution, that could cause docs to end up in some weird state in the release branch.